Sources/StreamChat/WebSocketClient/EventMiddlewares/UserWatchingEventMiddleware.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UserWatchingEventMiddleware.swift
+//  StreamChat
+//
+//  Created by Bahadir Oncel on 05/03/2021.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
```

Comment:
Seems like @DominikBucher12's fix broke the formatter 🤷‍♂️

Reply:
o.m.g...